jetstream/pages-ranking-experiment-revised.toml

```diff
@@ -1,18 +1,88 @@
 [metrics]
-weekly = ["history_clicks", "bookmark_clicks", "search_suggestion_clicks"]
-overall = ["history_clicks", "bookmark_clicks", "search_suggestion_clicks"]
+weekly = ["history_impressions", "bookmark_impressions", "search_suggestion_impressions",
+          "history_clicks", "bookmark_clicks", "search_suggestion_clicks"]
+overall = ["history_impressions", "bookmark_impressions", "search_suggestion_impressions",
+           "history_clicks", "bookmark_clicks", "search_suggestion_clicks"]
+
+
+[metrics.history_impressions]
+select_expression = "SUM(IF(product_result_type = 'history', num_impressions, 0))"
```

Comment:
You can count impressions from the existing `urlbar_events_unnested_results` data source using
```sql
COUNT(DISTINCT IF(is_terminal AND result.product_result_type = 'history', event_id, NULL))
```

eg. https://github.com/mozilla/metric-hub/blob/main/jetstream/cross-platform-enhanced-suggest.toml#L28